container-engine-lib/README.md

```diff
@@ -1,6 +1,6 @@
 Container Engine Lib
 ====================
-This library abstracts away interactions with the container engine (be it Docker or Kubernetes) via a `KurtosisBackend` interface. Users should call `GetLocalDockerKurtosisBackend` (or the Kubernetes equivalent when it exists).
+This library abstracts away interactions with the container engine (be it Docker or Kubernetes) via a `KurtosisBackend` interface. Users should call `GetDockerKurtosisBackend` (or the Kubernetes equivalent when it exists).
```

Comment:
I think `(or the Kubernetes equivalent when it exists)` is a reminiscence of when kubernetes was a backend plugin. 
And quickly searching the code I can't find an `GetKubernetesDockerBackend` anywhere, so what is this "equivalent" we're talking about here? Since you're touching this line, might be worth cleaning it up a little bit

Reply:
This is the asymmetry I was referring to last week with no `backend_creator` for K8S but there are similar functions of the form `New*KubernetesKurtosisBackend` which can be used for the K8S backend.  I updated the README.